playbooks/roles/edxapp/tasks/deploy.yml

```diff
@@ -423,3 +423,14 @@
   tags:
     - manage
     - deploy:platform
+
+- name: Create OAuth credentials
+  shell: >
+    {{ edxapp_venv_bin }}/python manage.py lms --setting=aws create_oauth2_client {{ item.url }} {{ item.redirect_uri }}
```

Comment:
@mrgnr I agree, there's no need to use `aws_appsembler` because this is a good thing to push upstream.